### DISCUSSION TITLE
How to display all files and folders on the server...

### DISCUSSION THREAD

**Discussion by afoster1955** [DirectoryLister/DirectoryLister#1106] [Help & Support]
I would like to use Directory Lister to show all the files and folders on the server...right now it is only showing the files in the folder where I have uploaded the files.  Perhaps that is not possible?

**Comment by SmartPhoneLover** [comment]
What I have done to allow showing custom locations, outside of what I could upload to the Directory Lister dir, is creating links as read-only, and place them in the root of webserver app. In other words, create links as read-only and place them where the directory Lister files are stored.
